src/valkey-benchmark.c

```diff
@@ -1511,8 +1534,15 @@ int parseOptions(int argc, char **argv) {
 #endif
         "";
 
+    rdma_usage =
+#ifdef USE_RDMA
+        " --rdma             Establish a RDMA connection.\n"
```

Comment:
Does it work with clusters?

Is guess it could work if RDMA port = TCP port, so CLUSTER SLOTS returns a port that can be used for RDMA for each node. Is this right?

Reply:
RDMA port must be as same as TCP port in the cluster, this is the limitation under current version. Additional fields may be needed, let's leave it as a TODO topic in the future.

Reply:
Yes, maybe we don't even need to fix it ever. Just a note somewhere in the documentation would be good, or a comment in the valkey.conf.